ton_client/client/src/crypto/hdkey.rs

```diff
@@ -77,13 +77,13 @@ pub fn hdkey_secret_from_xprv(
 
 #[derive(Serialize, Deserialize, ApiType)]
 pub struct ParamsOfHDKeyPublicFromXPrv {
-    /// serialized extended private key
+    /// Serialized extended private key
     pub xprv: String,
 }
 
 #[derive(Serialize, Deserialize, ApiType)]
 pub struct ResultOfHDKeyPublicFromXPrv {
-    /// public key
+    /// Public key - unprefixed 0-padded to 64 symbols hex string
```

Comment:
Just `64 symbols hex string` is enough. No need to write about prefix and padding.